src/ostorlab/serve_app/types.py

```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
 
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_apk_file = graphene.List(OxoAndroidFileAssetInputType)
```

Comment:
The we are missing android_aab_file

Reply:
the name is just to align with the yaml file, in the db its named android_file and we store there all android files (apk, aab, xapk)

the logic to separate between them is in runScan : 

![image](https://github.com/Ostorlab/oxo/assets/144013278/b1506966-35f6-4bf0-a1ec-898bbfba19a3)

Reply:
added